health_check/backends.py

```diff
@@ -43,9 +46,26 @@ def add_error(self, error, cause=None):
             logger.error(str(error))
         self.errors.append(error)
 
-    def pretty_status(self):
+    def add_warning(self, warning, cause=None):
+        if isinstance(warning, HealthCheckException):
+            pass
+        elif isinstance(warning, str):
+            msg = warning
+            warning = HealthCheckException(msg)
+        else:
+            msg = _("unknown error")
+            warning = HealthCheckException(msg)
+        if isinstance(cause, BaseException):
+            logger.exception(str(warning))
```

Comment:
I don't know if you really want to log at exception level here. I would go for warning.